pipeline/pipelines/code_generation_pipeline.py

```diff
@@ -35,6 +35,11 @@
                         'pypi_pwd',
                         'publish_env']
 _PYTHON_PUB_ENVS = ['dev', 'test', 'staging', 'prod']
+_JAVA_PUB_REQUIRED = ['maven_repo_url',
+                      'maven_uname',
+                      'maven_pwd',
```

Comment:
I really don't like having there prefixed parameters separately.
This should be consolidated with pypi params, and named simply as 'server_url', 'user_name', and 'password'.